frictionless/server/table/export.py

```diff
@@ -0,0 +1,23 @@
+from typing import Optional
+from pydantic import BaseModel
+from fastapi import Request
+from ...project import Project
+from ..router import router
+
+
+class Props(BaseModel):
+    session: Optional[str]
+    path: str
```

Comment:
It seems we can just accept `source` (path) and `target` (path) as we only use format to merge it with the name at the beginning (and the client can merge them by itself)